web_app/src/app/form1/steps/step1.tsx

```diff
@@ -93,6 +100,30 @@ const Step1 = forwardRef((props: Props, ref) => {
         props.step1DatatoSend(step1Data);
     };
 
+    const [tooltipContent, setTooltipContent] = useState<{ [key: string]: JSX.Element | string }>({});
+
+    useEffect(() => {
+        setTooltipContent({
+            tooltipContent1: "Enter first 8 numbers with the dash. Don't enter the digit in the grey box. Prohibition numbers start with 00, 21, 30 or 40.",
+            tooltipContent2: (
+                <img 
+                    src="/assets/images/License Seized.png"
+                    width={380}
+                    height={145}
+                />
+            ),
+            tooltipContent3: (
+                <img src="/assets/images/Prohibition Period and Type.png"
+                    width={280}
+                    height={180}
+                    alt="Info" />),
+            tooltipContent4: (
+                <p>Enter first 8 numbers with the dash.Don&apos;t enter the digit in the grey box. Prohibition numbers start with 00, 21, 30 or 40.</p>
```

Comment:
Enter "the" first 8 numbers. 
"digits" in the grey box.

Reply:
these were copied from the text already there not by me but sure can make the changes

Reply:
I looked at this, the same phrase is used on the page and the tooltip in form1 and form3 changing it here in these 2 tooltip cases seems not correct if not changing others. So for this best approach is to reach out to the client if this is what they want or not. If the client agrees I'll make the change everywhere.